stylix/droid/fonts.nix

```
@@ -0,0 +1,29 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  mkFont =
+    font:
+    pkgs.runCommand "stylix-font-${font.package.name}"
+      {
+        FONTCONFIG_FILE = pkgs.makeFontsConf { fontDirectories = [ font.package ]; };
+      }
+      ''
+        font=$(
+          ${lib.getExe' pkgs.fontconfig "fc-match"} \
+          ${lib.escapeShellArg font.name} \
+          --format %{file}
+        )
+        ln -s "$font" "$out"
```

Consider the following improvements:

```suggestion
        font="$(
          ${lib.getExe' pkgs.fontconfig "fc-match"} \
          ${lib.escapeShellArg font.name} \
          --format %{file}
        )"

        ln --symbolic "$font" "$out"
```

AFAIK, `cp` is the most reliable inside derivations. Consider using `cp` instead of `ln`.